css/src/components/message.scss

```diff
@@ -58,4 +60,16 @@ $message-border: $border-width solid $border !default;
 			background-color: $message-sender-background-color;
 		}
 	}
+
+	&.message-size-sm {
+		font-size: $message-font-size-sm;
+
+		.persona-details .persona-name {
+			font-size: $message-font-size-sm;
+		}
+
+		.message-content {
+			padding: 0.875rem;
```

Comment:
If the padding on `message-content` was set up in relative units such as `em`, the only thing you would need to do to add a size variation is to update the `font-size` on `.message-sm`. That way everything scales perfectly while keeping the proportions. Please update the `message-content` padding's value to `em` and remove these lines.

Check out the example here: https://github.com/microsoft/atlas-design/blob/main/css/src/components/persona.scss#L51-L59C3